internal/pkg/template/templates/environment/cf.yml

```diff
@@ -742,3 +798,15 @@ Outputs:
   LastForceDeployID:
     Value: {{quote .ForceUpdateID}}
     Description: Optionally force the template to update when no immediate resource change is present.
+{{- if .VPCConfig.Flowlogs }}
+  FlowLogID:
+    Value: !Ref Flowlog
+    Description: VPC Flow Log to capture information about the IP traffic going to and from network interfaces in your VPC.
+    Export:
+      Name: !Sub ${AWS::StackName}-FlowlogID
+  FlowLogRoleID:
+    Value: !Ref  Flowlogrole
+    Description: An IAM Role for EC2 to publish flow logs to a CloudWatch Logs log group  
+    Export:
+      Name: !Sub ${AWS::StackName}-FlowlogRoleID
+{{- end}}
```

Comment:
We put things into `Outputs` section mostly when we think they'd be useful for users to reference from the other stacks. In this case we are probably fine not outputing + exporting them because it's unlikely that the users will need to reference flog logs anywhere else. What do you think?

Reply:
Also it's a one-way door decision for outputting them whereas if we don't output for now we can add it to the output ad hoc.